src/app/workspace/resource/resource-instance-form/resource-instance-form.component.ts

```diff
@@ -234,14 +245,19 @@ export class ResourceInstanceFormComponent implements OnInit, OnDestroy {
                         response.ontologies = response.ontologies.filter(onto => onto.attachedToProject !== Constants.SystemProjectIRI);
 
                         this.ontologiesMetadata = response;
+
+                        // notifies the user that the select project has got 0 data model yet defined.
```

Comment:
notifies the user that the selected project does not have any data models defined yet.

Reply:
fixed in 3717fc5